lib/contentTypeParser.js

```diff
@@ -93,19 +93,21 @@ ContentTypeParser.prototype.existingParser = function (contentType) {
 }
 
 ContentTypeParser.prototype.getParser = function (contentType) {
-  /* eslint-disable no-var */
-  for (var i = 0; i < this.parserList.length; i++) {
-    if (contentType.indexOf(this.parserList[i]) > -1) {
-      const parser = this.customParsers[this.parserList[i]]
+  // eslint-disable-next-line no-var
+  for (var i = 0, len = this.parserList.length; i !== len; ++i) {
+    const parserName = this.parserList[i]
+    if (contentType.indexOf(parserName) > -1) {
```

Comment:
```suggestion
  for (var i = 0, len = this.parserList.length; i !== len; ++i) {
    const parserName = this.parserList[i]
    if (contentType.indexOf(parserName) > -1) {
```

+ I think the caching of `parserName` is a fine optimization.
+ The pre-calculation of `len` is unnecessary. V8 long ago added this internally.
+ I don't particularly care for either form of the increment operator. I prefer `i += 1`.

Reply:
+1

Reply:
+1